frontend/internal-packages/agent/src/qa-agent/types.ts

```diff
@@ -8,7 +8,7 @@ export const testcaseSchema = v.object({
   requirement: v.string(),
   title: v.string(),
   description: v.string(),
-  dmlOperations: v.array(dmlOperationSchema),
+  dmlOperation: dmlOperationSchema,
```

Comment:
The decision to split the QAAgent into separate runs led to the creation of a large number of test cases for a single requirement. One of the moves we made to solve this was to generate all requirements, test cases and dmlOperation one by one.